jersey228/src/main/java/org/pac4j/jax/rs/grizzly/pac4j/GrizzlyJaxRsContext.java

```diff
@@ -22,8 +25,8 @@
 
     private final Request request;
 
-    public GrizzlyJaxRsContext(Providers providers, ContainerRequestContext requestContext,
-            SessionStore sessionStore, Request request) {
+    public GrizzlyJaxRsContext(Providers providers, ContainerRequestContext requestContext, SessionStore sessionStore,
```

Comment:
Same feedback as for the `JaxRsContent`. The `SessionStore` should be treated separately.